src/components/Filters.js

```diff
@@ -13,16 +24,38 @@ const Filters = () => {
     <ul className="sidebar">
       <span id="sidebar__link_h">Genres of films:</span>
       <li id="sidebar__link ">
-        <a href="#" className="sidebar__link 1 active" data-link="main">
+        <Link
+          to="/moviesApp-React"
+          className="sidebar__link top active"
+          data-link="v2.2/films/top?type=TOP_100_POPULAR_FILMS"
+          onClick={(e) => {
```

Comment:
Move handlers to standalone functions if there are more than one row maximum two rows. It very hard to read and understand long ones in jsx code.